src/main/java/ca/corefacility/bioinformatics/irida/processing/FileProcessor.java

```diff
@@ -40,12 +40,12 @@
 	/**
 	 * This method asks the file processor whether it should act on this file.
 	 * The processor may have some settings that would not run on certain files.
-	 * 
-	 * @param sequencingObjectId
-	 *            the {@link SequencingObject} id to check
+	 *
+	 * @param sequencingObject
+	 *            the {@link SequencingObject} to check
 	 * @return true if the processor should act on the file
 	 */
-	public default boolean shouldProcessFile(Long sequencingObjectId) {
+	public default boolean shouldProcessFile(SequencingObject sequencingObject) {
```

Comment:
This wasn't used anywhere except tests yet. It works perfectly for this use case, but I changed it to accept the object instead of the id.  That made sense seeing it was being read in the `DefaultFileProcessingChain` right after anyway.

Reply:
Looks good, thanks